### DISCUSSION TITLE
Result List Model binding

### DISCUSSION THREAD

**Discussion by jasenf** [scalar/scalar#4346] [Q&A]
Hi all --

This may sound nit picky, but is it possible to have the Scalar page show the actual object type of the list?  Am I marking the controller method properly or is there a different way it should be done?   When the user sees the object[] they have to expand it, look at the attributes and try to determine which Model is actually being used.  I'd prefer the items in orange to show PeoplePagedResultDto and PersonDto[] respectively if possible.

<img width="965" alt="image" src="https://github.com/user-attachments/assets/4965c827-dafc-4c3c-b483-bdfdc371d3c8" />

I think the OpenAPI json file shows the bindings properly

<img width="386" alt="image" src="https://github.com/user-attachments/assets/62f97da6-dfac-424e-a816-be9dfe9ed75d" />

**Comment by hanspagel** [comment]
True! We plan to add this. It’s way harder than it seems though. 😅

**Reply by jasenf** [reply]
Ok, no worries.  Just wanted to see if it was currently possible.  Thanks for the quick response.
